src/jit/lsra.cpp

```diff
@@ -188,6 +188,38 @@ unsigned LinearScan::getWeight(RefPosition* refPos)
     return weight;
 }
 
+//-------------------------------------------------------------------------
+// MarkSiblingRefPosIfAny: Marks the sibling ref pos associated with the
+// given ref pos as non-RegOptional and also removes the mapping between
+// the two ref positions.
+//
+// Arguments:
+//    refPos   -   ref position that is marked as 'Alloc if profitable'
+//
+// Returns:
+//    Nothing.
+void LinearScan::MarkSiblingRefPosIfAny(RefPosition* refPos)
+{
+    assert(refPos->AllocateIfProfitable());
+
+    // It is possible that refPos mapping table was not created
+    // if there were no mappings added (e.g. nodes have max
+    // one reg optional operand).
+    if (refPosToSiblingRefPosMap != nullptr)
+    {
+        RefPosition* siblingRefPos = nullptr;
+        bool found = refPosToSiblingRefPosMap->Lookup(refPos, &siblingRefPos);
+        if (found)
+        {
+            assert(siblingRefPos != nullptr);
+            assert(siblingRefPos->AllocateIfProfitable());
+            siblingRefPos->setAllocateIfProfitable(0);
```

Comment:
For clarity, this method should take a bool parameter, and this should be `setAllocateIfProfitable(false)`.